examples/devmode.yaml

```diff
@@ -3,3 +3,9 @@ system:
     plugins:
         -
           name: aeplugin_dev_mode
+          config:
+            prefunded_accounts: []
```

Comment:
is there a reason we have this over `.genesis/accounts_test.json`?

Reply:
I just wanted something there to illustrate the plugin config item. :)

Reply:
The empty list is the schema default, i.e. no prefunded accounts.
This could of course be discussed.

Reply:
was just wondering why there is the need for both options :D